test/Faker/Provider/PhoneNumberTest.php

```diff
@@ -11,10 +11,10 @@
  */
 final class PhoneNumberTest extends TestCase
 {
-    public function testPhoneNumberFormat()
+    public function testE164PhoneNumberFormat()
     {
         $number = $this->faker->e164PhoneNumber();
-        self::assertMatchesRegularExpression('/^\+[0-9]{11,}$/', $number);
+        self::assertMatchesRegularExpression('/^\+[1-9][0-9]{10,}$/', $number);
```

Comment:
@krsriq wouldnt it be better here to run x number of iterations with unique? Since this way we actually just test a single number right?

Reply:
If it's wanted I can change this. It's the way it was tested before.
The "official" regex would be `^\+[1-9]\d{1,14}$` anyway, so if I change the tests I would also change the regex to match the e164 standard and not what we generate - as in theory numbers can be shorter than CC+10digits and also have an upper bound.

Reply:
Regarding the regex: we could also change the generation logic to randomize the number of digits after CC.
But I haven't found a way to do so in an easy way as we would have to randomize the number of `#` like `str_repeat('#', mt_rand(8, 12))` which doesn't work with the format is.
Somewhere I've seen that `numerify()` somewhen supported `!` to put in a digit or nothing but this doesn't work anymore. 🤔

Reply:
I've updated the regex assertions to test for E164 standard and not for the current generator.
https://github.com/FakerPHP/Faker/pull/264/commits/a159368025785715dadc754406ecd201afee7aa7
This way the test can remain the same if somewhen someone wants to improve the generator even more and/or randomize the number of digits after CC.

Reply:
@pimjansen I've checked some other tests and this is done in some other phone tests already for the "normal" phone number.
So I've added the `for` loops - 1000 runs for the large one to get a lot of different CC prefixes and 10 per country provider as these only have a single format.
https://github.com/FakerPHP/Faker/pull/264/commits/ac76e295bb83de5acbd704c30974a473ca28bec0

Reply:
> @krsriq wouldnt it be better here to run x number of iterations with unique? Since this way we actually just test a single number right?

I don't think unique adds any value here, since it's already highly unlikely that a number with ten digits is generated twice (even with 1000 iterations).